### ISSUE TITLE
error installing new version 2.1.4

### ISSUE THREAD

**Issue by derEremit** [atom-community/autocomplete-plus#260]
probably something wrong with the build script

```
npm ERR! node v0.10.35
npm ERR! npm  v2.3.0
npm ERR! path /var/lib/jenkins
npm ERR! code EACCES
npm ERR! errno 3

npm ERR! Error: EACCES, mkdir '/var/lib/jenkins'
npm ERR!  { [Error: EACCES, mkdir '/var/lib/jenkins'] errno: 3, code: 'EACCES', path: '/var/lib/jenkins' }
```

**Comment by joefitzgerald** [comment]
This is a permissions error with yiur build user, and has nothing to do with this package. Make sure your user's home directory is set to somewhere writeable for that user.

**Comment by biffen** [comment]
I'm seeing the same error, and I most definitely have write permissions in my home directory.

According to the log it's trying to create a `jenkins` directory in `/var/lib/`. I can't find any reference to it in this package, has v2.1.4 got any changes in dependencies?

**Comment by derEremit** [comment]
just updated installed ~20 packages on 3 different machines.
Autocomplete plus 2.1.4 same error on all 3 machines, all other packages installed just fine.
and I don't have anything with jenkins on my local machines.

**Comment by joefitzgerald** [comment]
What version of Atom?

**Comment by joefitzgerald** [comment]
FWIW we don't even use Jenkins on the project - we use Travis and Appveyor. I suspect this is an issue with Atom `0.178.0` or later.

**Comment by derEremit** [comment]
Sorry, should have provided that information:
atom 0.178.0 from webupd8 ppa
ubuntu 14.04 64bit

**Comment by derEremit** [comment]
sorry, i now get the error when installing other packages too,
really seems to be an atom problem

**Comment by joefitzgerald** [comment]
See: https://github.com/atom/atom/issues/5492

**Comment by joefitzgerald** [comment]
I believe this is fixed in `0.179.0`.
